services/wallet/router/router.go

```diff
@@ -317,24 +317,10 @@ func arrayContainsElement[T comparable](el T, arr []T) bool {
 	return false
 }
 
-func arraysWithSameElements[T comparable](ar1 []T, ar2 []T, isEqual func(T, T) bool) bool {
```

Comment:
@dlipicar what I had in mind with this function was to compare `DisabledFromChainIDs` and `DisabledToChainIDs` arrays, cause when client sends mainnet chain enabled only it sends for both arrays `[10, 42161], but now since they are converted to "enabled chains" we don't need it I guess.

Reply:
hmm not sure if you were planning to use that function in the future (you can re-introduce it then), it didn't make sense where it was used before since we had this check right before calling this 
![image](https://github.com/user-attachments/assets/04631af8-1c02-488f-9817-19565330c1ed)